Profile updates (#15)
canonical/landscape-documentation: docs/how-to-guides/repository-mirrors/manage-repositories-in-the-web-portal.md

## Create a repository profile and associate computers to the profile

> A Repository Profile in Landscape is useful for updating repository configurations. When a machine is associated with a Repository Profile, the repository configurations are applied one time. Repository Profiles do not perform ongoing monitoring of repository configurations.

Reviewer: ```suggestion
A repository profile in Landscape is useful for updating repository configurations. When a machine is associated with a repository profile, the repository configurations are applied one time. Repository profiles don't perform ongoing monitoring of repository configurations.
```
